src/components/Header/Header.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
-import SideBar from './SideBar';
+import ConnectedSideBar from './SideBar';
 
 export default class Header extends Component {
   render() {
-    return <SideBar />;
+    return <ConnectedSideBar />;
```

Comment:
If there are no plans to have more markup in header, than this component is obsolete and we can use Sidebar directly